src/GraphQL.Tests/Validation/DefaultValuesOfCorrectTypeTests.cs

```diff
@@ -23,7 +23,8 @@ public void required_variables_without_default_values()
               query RequiredValues($a: Int!, $b: String!) {
                 dog { name }
               }
-            ");
+            ",
+            "{ \"a\": 1, \"b\": \"\" }");
```

Comment:
I had to add variables for all successful tests to keep them that way, since getting the variables is now part of the validation. Previously, the validation completed successfully, and the error would have occurred immediately after that at the time of building `ExecutionContext`.